cgsmiles/drawing.py

```diff
@@ -164,6 +165,14 @@ def draw_molecule(graph,
     elif colors is None:
         colors = {node: ELE_TO_COLOR.get(ele, DEFAULT_COLOR) for node, ele in nx.get_node_attributes(graph, 'element').items()}
 
+    # compute angle for axis alignment
+    if align_with == 'diag':
+        align_with = np.array(diagonal)
+    elif align_with == 'x':
+        align_with = np.array([1,0])
+    elif align_with == 'y':
+        align_with = np.array([0, 1])
```

Comment:
nit: dictionary?

Reply:
you missed the obvious if `align_with` is an array the if statements will fail ;)

Reply:
Ahh just check if the type is a string and the get the dict. Keeping it simple